zchronod/logger/src/lib.rs

```diff
@@ -0,0 +1,62 @@
+use {
+    lazy_static::lazy_static,
+    std::sync::{Arc, RwLock},
+};
+
+// global chrono_log config
+lazy_static! {
+    static ref CLOGGER: Arc<RwLock<env_logger::Logger>> =
+        Arc::new(RwLock::new(env_logger::Logger::from_default_env()));
+}
+
+struct zChronodLogger {}
+
+impl log::Log for zChronodLogger {
+    fn enabled(&self, metadata: &log::Metadata) -> bool {
+        CLOGGER.read().unwrap().enabled(metadata)
+    }
+
+    fn log(&self, record: &log::Record) {
+        CLOGGER.read().unwrap().log(record);
+    }
+
+    fn flush(&self) {}
+}
+
+
+
+/// * `logfile_path`: path to store log file
+/// * `filter`: log level
+///
+/// # Examples
+/// init_chrono_logger_with_path("./xx/chronod.log", "debug")
+pub fn init_zchronod_logger_with_path(logfile_path: &str, filter: &str) {
+    let file = std::fs::OpenOptions::new()
```

Comment:
truncate and write to the same log file maybe not a suitable way for a long-time running daemon process, usually we need these functionalities:
- new log file every startup
- save & new a logfile when too much content
- record timestamp in log file
- save $ create a new logfile when a new day start